clio-server/src/main/scala/org/broadinstitute/clio/server/service/SchemaService.scala

```diff
@@ -0,0 +1,188 @@
+package org.broadinstitute.clio.server.service
+
+import java.time.OffsetDateTime
+
+import io.circe._
+import org.broadinstitute.clio.transfer.model.TransferReadGroupV1QueryOutput
+import shapeless._
+import shapeless.labelled._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.reflect.runtime.universe.{Type, TypeTag, typeOf, typeTag}
+
+/**
+  * Generate JSON schema from case classes such as the QueryOutputs.
+  *
+  * JsonSchema uses the same shapeless pattern as the
+  * org.broadinstitute.clio.util.generic.FieldMapper
```

Comment:
It looks like scaladoc links are done using `[[` `]]`, e.g. `[[org.broadinstitute.clio.util.generic.FieldMapper]]`